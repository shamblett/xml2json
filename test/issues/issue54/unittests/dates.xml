<?xml version="1.0" encoding="utf-8"?>
<dates version="1.0" xmlns:ding="http://zanstra.com/ding">
  This should be ignored, as usual.
  <birthdate>2006-11-14</birthdate>
  <appointment>2006-11-14T12:30</appointment>
  <appointment>2006-11-14T12:30:00</appointment>
</dates>
