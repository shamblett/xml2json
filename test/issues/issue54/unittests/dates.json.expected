{"dates":{"birthdate":"2006-11-14","appointment":"2006-11-14T12:30","appointment":"2006-11-14T12:30:00"}}
